### DISCUSSION TITLE
GCode loads and immediately unloads

### DISCUSSION THREAD

**Discussion by 777Dion** [winder/Universal-G-Code-Sender#2344] [General]
Hi,

I created a NC file with fusion 360. Then I tried to open it with UGS. It loads to the point where I see the NC file flash on the visualizer but it immediately unloads. As a result I cannot click the play button on UGS.

I tried to download a gcode file from the internet and I get the same error, I cannot open it with UGS.

When i design something within UGS like a rectangle then that works fine. There seems to be a problem loading NC files.

Please help, what can I do to resolve this problem.

**Comment by breiler** [comment]
Please retry loading a file, then post the log file from:
`C:/Users/[your username>]/AppData/Roaming/ugsplatform/var/log/messages.log`

**Reply by 777Dion** [reply]
[messages.log](https://github.com/winder/Universal-G-Code-Sender/files/13021959/messages.log)

Thank for your feedback. The message.log exceeds 65536 characters and is not allowing me to paste it into comments. I've tried to attach it herein this reply.

**Reply by breiler** [reply]
> I've tried to attach it herein this reply.

Thanks, I prefer it like that!

To fix this I think that you'll need to remove the cache directory manually. But before we try that I might have a automatic fix for this in the latest nightly build. If you would consider testing that I would appreciate it!

Download the latest nightly build from here, unpack and start it as usual: 
https://github.com/winder/Universal-G-Code-Sender#downloads

If this doesn't work you will need to delete the folder `C:\Users\Dion\AppData\Local\ugsplatform\Cache\var\cache` and then start the application.

**Reply by 777Dion** [reply]
I downloaded and tested the latest nightly build at the download link that you provided above and everything is working perfectly fine now. :) 👍 

Thanks a million for your help, the problem mentioned above has been resolved. :)

**Reply by breiler** [reply]
Cool, thanks for the feedback. This means that we might have a viable solution for this problem.
